src/plugins/discover/public/application/helpers/get_sharing_data.test.ts

```diff
@@ -46,10 +46,8 @@ describe('getSharingData', () => {
         ],
         "searchRequest": Object {
           "body": Object {
-            "_source": Object {
-              "includes": Array [],
-            },
-            "docvalue_fields": Array [],
```

Comment:
This disappeared from the snapshot since we stopped using `docvalue_fields` by default.